src/components/categories/categories.css

```diff
@@ -0,0 +1,29 @@
+.left-bar {
+  position: absolute;
+  left: auto;
+  width: 50px;
+  height: 10px;
+  line-height: 40px;
+  padding-left: 5%;
+  color: #0070f3;
+  font-weight: bolder;
+  font-size: 17px;
+  text-transform: uppercase;
+  text-align: left;
+}
+.left-bar button {
+  text-align: center;
+  white-space: nowrap;
+}
+.left-bar button:hover {
+  background-color: #d0d8e2;
+}
```

Comment:
On hover, I'd add something which change the colour of the text or increase slightly the font-size. The grey bg-color is not "subtil".

Reply:
this is the hover effect

![Screenshot (9)](https://user-images.githubusercontent.com/20721985/103413970-7d9adf00-4b84-11eb-8f61-f21a32df9d3d.png)

Reply:
It is better. Can you make it a darker blue instead ? Something like this color #074088
The rest is good

Maybe another improvement. Can we please order the tags in an alphabetical order ?